### ISSUE TITLE
Support V2 test format in the single-page view of a test plan

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at-app#745]
#### Overview

Change the content shown on the test plan version view page for versions of test plans built using the V2 format.

Note:
* It is essential the system supports viewing versions of test plans built from both V1 and V2 test plan formats. 
* The [V2 format](https://github.com/w3c/aria-at/wiki/Test-Format-Definition-V2) contains a column for `presentationNumber` in tests.csv. That number specifies the sequence of test presentation.

#### Changes to the test plan view page

All but one of the changes to the build/review files listed in w3c/aria-at#977 also need to be made to the test plan review page. Ignore the first change related to the level 1 heading. Make all subsequent changes listed in w3c/aria-at#977, starting with the label for the radio filters.

**Comment by mcking65** [comment]
This issue says the fix is in the sandbox. However, the sandbox data doesn't have any v2 formatted test plans as far as I can tell.

I checked in staging, and the content of the test plan pages is incorrect. This is of sufficient concern that we should either:
1. Block deployment until resolved.
2. Deploy the staging environment only if a fix for this could be deployed in a subsequent fast-follow deployment that is less than a week later.

To validate, I looked at the command button test plan:
* [Build preview of command button test plan in V2 format in the aria-at repo](https://aria-at.netlify.app/review/command-button)
* [View of the same version of command button test plan from the staging server](https://aria-at-staging.w3.org/test-review/68023)

When comparing, note the following problems with the staging server view of the test plan:
1. Following the instructions in each test, there is a heading 5 "success criteria" followed by some obsolete V1 content.
2. The headings for each command and assertion tables are not presented.
3. Under the heading "About This Test Plan", the title of the latest commit is not a link; it should be a link just like it is on the test plan version history page. Editorial note: I think it would be better if that list item had content: "Latest commit: [title of latest commit]().
4. There is an information architecture problem that does not appear to be covered by the original specification. The tests are under an H2 of "Supporting Documentation" instead of under the H2 "Tests". The "Supporting Documentation" heading should be moved before the ""Tests" heading. The resulting H2 order would then be: "About This Test Plan", "Supporting Documentation", "Tests". Then, each individual test will be an H3 under the H2 of "Tests".
